### DISCUSSION TITLE
Macro Placement

### DISCUSSION THREAD

**Discussion by BaranErdo** [The-OpenROAD-Project/OpenROAD-flow-scripts#1205] [Q&A]
Hello,

I was working with openroad to implement our project. It worked at first but than I wanted to add the macro placement too however the process gets stuck at 2_4mplace. We are using srams generated with OpenRAM. I defined the lefs, libs and gds files for the sram macro is there any chance that I have to do to make the program work or is there  another problem?

 ./logs/sky130hd/core_v_mini_mcu/base/2_4_mplace.log
OpenROAD v2.0-5852-g2a9ef5fa1 
This program is licensed under the BSD-3 license. See the LICENSE file for details.
Components of this program may be licensed under more restrictive licenses which must be honored.
[INFO MPL-0005] Found 16 macros.
[INFO MPL-0009] West pins 113.
[INFO MPL-0009] East pins 112.
[INFO MPL-0009] North pins 166.
[INFO MPL-0009] South pins 166.

**Comment by vijayank88** [comment]
Please attach a test case by running `make macro_place_issue` to debug the issue. @BaranErdo

**Reply by BaranErdo** [reply]
Thank you for the suggestion. I did what you said and runned the generated run_me file, however I still have the same problem.

**Reply by rovinski** [reply]
@BaranErdo the testcase file is for you to attach here so that we can help you debug it. This should be a GitHub issue and not a discussion.

**Reply by BaranErdo** [reply]
I posted the testcase file and the problem to the issues. Thank you for the advice @rovinski
